### PR TITLE
Add note about VMware backup.

### PR DESCRIPTION
**Pull request by jsafrane**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1547156

This should be cherry picked into all older releases!

### CODE REVIEW

File: install_config/configuring_vsphere.adoc [hunk 1]
```diff
@@ -235,3 +235,13 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
 [[vsphere-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* so
+it can freely attach and detach the volume any node in the cluster. As
```
Comment (tmorriso-rh) [inline install_config/configuring_vsphere.adoc]: The first sentence is not clear. Should it be "{product-title} provisions new volumes as independent persistent disks to freely attach and detach the volume on any node in the cluster."

File: install_config/configuring_vsphere.adoc [hunk 2]
```diff
@@ -235,3 +235,13 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
...
+a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to backup these volumes using snapshots].
+
```
Comment (tmorriso-rh) [inline install_config/configuring_vsphere.adoc]: s/to backup/to back up

File: install_config/configuring_vsphere.adoc [hunk 3]
```diff
@@ -235,3 +235,13 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
...
+It's recommended to stop application that uses a PV, clone its persistent disk
+and start the application to take a backup of a volume.
```
Comment (tmorriso-rh) [inline install_config/configuring_vsphere.adoc]: Suggest changing to "It is recommend to stop the application using the PV, clone its persistent disk, and restart the application to create a backup of the volume."

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL for peer review. Thanks.

File: install_config/configuring_vsphere.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install_config/configuring_vsphere.adoc]: I like your edit.

File: install_config/configuring_vsphere.adoc (hunk 3 above)
Comment (kalexand-rh) [reply install_config/configuring_vsphere.adoc]: s/using/that uses

**Review by kalexand-rh — COMMENTED** [review]
I have one suggestion, but with if your edits are incorporated, this LGTM.

File: install_config/configuring_vsphere.adoc (hunk 1 above)
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Fixed

File: install_config/configuring_vsphere.adoc (hunk 2 above)
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Fixed

File: install_config/configuring_vsphere.adoc (hunk 3 above)
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: I added steps 4 and 5:

1. Stop the application using the PV.
2. Clone the persistent disk.
3. Restart the application.
4. Create a backup of the cloned disk.
5. Delete the cloned disk.

File: install_config/configuring_vsphere.adoc [hunk 4]
```diff
@@ -235,3 +235,17 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
...
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* to
+freely attach and detach the volume on any node in the cluster. As a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to back up these volumes that uses snapshots].
+
+To create a backup of PVs:
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Removed (replaced with ==Procedure)

File: install_config/configuring_vsphere.adoc [hunk 5]
```diff
@@ -235,3 +235,17 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
...
+
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Rewritten to `== Procedure`

File: install_config/configuring_vsphere.adoc [hunk 6]
```diff
@@ -235,3 +235,17 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
 [[vsphere-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* to
+freely attach and detach the volume on any node in the cluster. As a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to back up these volumes that uses snapshots].
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Fixed

File: install_config/configuring_vsphere.adoc [hunk 7]
```diff
@@ -235,3 +235,18 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
...
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* to
+freely attach and detach the volume on any node in the cluster. As a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to back up volumes that use snapshots].
+
+[discrete]
```
Comment (tmorriso-rh) [inline install_config/configuring_vsphere.adoc]: Can you remove this heading? We don't need the `Procedure` heading, but you could add an introductory sentence, such as "To back up PVs that use snapshots:"

Comment (jsafrane) [reply install_config/configuring_vsphere.adoc]: Heading removed, restored "To create a backup of PVs:".

**Comment by vikram-redhat** [comment]
Hey @tmorriso-rh - was this supposed to be picked for 3.6 as well?

**Comment by tmorriso-rh** [comment]
@vikram-redhat No, just 3.7 - 3.10. I have removed the 3.6 label. Thanks.

**Pull request merged**